### ISSUE TITLE
magick command to make icons does not exist on Ubunut 20.04 - but I have ImageMagick installed

### ISSUE THREAD

**Issue by terremoth** [c-smile/sciter-js-sdk#300]
The "magick" command does not exist, however the "convert" command (used by imagemagick to convert images) exist.

ImageMagick can be easily installed on Ubuntu by simply `sudo apt install imagemagick`

![image](https://user-images.githubusercontent.com/11032344/160215009-fa7d83d7-4ee7-4ad4-9ec8-a805da8a4a78.png)

Look what quark setup tries to do:

![image](https://user-images.githubusercontent.com/11032344/160215196-6236859f-d041-453a-b4b6-0e72a3658240.png)

Now you could say "ah this can be solved by just creating an alias in your bashrc like `alias magick='convert'` right? 
No, even if I restart the application, does not work either. Seems like a bug, probably easy to correct.

**Comment by terremoth** [comment]
I solved (I think the problem manually) by changing quark/package.js file and changing the "magick" command to "convert"
Maybe you could do a variable with a check before if `convert` command exists and "magick" not, and use `convert`

However... It seems there is another problem:

![image](https://user-images.githubusercontent.com/11032344/160216856-13985cf4-05bf-4093-8754-048eeb2e2eb7.png)

I am trying, as you see, to generate an app for win64 and linux, and it shows this "winx64 FAILURE, no .dat file" and no Windows file (even the .exe) was generated.

**Comment by terremoth** [comment]
> Actually imagemagik is only used to create the icon for Windows executables. Both Linux and Mac do not have support for icons within the executable.

Of course, but in Ubuntu the Quark and the whole software won't work if this change is not made... The executable will never be generated, this is why this is a bug.

**Comment by 8ctopus** [comment]
I guess we can just ignore the whole imagemagik code when not on Windows. @terremoth Maybe you can create a PR?

**Comment by terremoth** [comment]
I think I can do better if I have time: make an "if convert command exists and imagick command does not, use convert command instead", and it will work, but the second problem will remain to create winx64 dat file.

**Comment by terremoth** [comment]
It would be good to create windows application directly from linux/ubuntus too, so it would need to correctly generate dat file and exe file

**Comment by 8ctopus** [comment]
Yes you're right.

**Comment by terremoth** [comment]
I can solve the first problem, but the second one I don't think I can figure out myself
